packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -166,13 +175,20 @@ export class VirtualGateway extends VirtualGatewayBase {
       props.listeners.forEach(listener => this.listeners.push(listener.bind(this)));
     }
 
+    if (props.backendDefaults) {
+      this.backendDefaults = this.addBackendDefaults(props.backendDefaults);
```

Comment:
I don't think this is a good method name. `add` should only be used if there can be multiples of the thing added. Since this doesn't seem to be the case here, I would go for something like `setBackendDefaults`.

Reply:
Updated the function name to `setBackendDefaults`